tests/test_selenium_app.py

```diff
@@ -2,13 +2,12 @@
 import random
 from threading import Thread
 
+from todoapp import app
+from todoapp import common
 import pytest
 from dotenv import find_dotenv, load_dotenv
 from selenium import webdriver
-
-import app
-import common
-from trello_request import TrelloRequest
+from todoapp.trello_request import TrelloRequest
```

Comment:
I get an import error when running it in docker without an `__init__.py` file in the todoapp folder.